website/src/pages/Why.vue

```diff
@@ -33,7 +33,7 @@ BasicLayout.Why
         <a href="https://gs.statcounter.com/browser-version-market-share/desktop/worldwide/">top 3</a> most popular browsers are ready to plug-in.
 
 
-      h2 Writing Scraper Scripts is Still too Complicated
+      h2 Writing Scraper Scripts Is too Complicated
```

Comment:
Why capitalized? Shouldn't that be lower case?

Reply:
Verbs should always be capitalized. And according to https://titlecaseconverter.com/, the word "too" should also be capitalized... just pushed an update.